src/modules/vtol_att_control/vtol_att_control_main.cpp

```diff
@@ -663,7 +668,19 @@ void VtolAttitudeControl::task_main()
 			_vtol_type->update_transition_state();
 		}
 
-		_vtol_type->fill_actuator_outputs();
+		if (_params.v19_vt_rolldir) {
+
+			// The mixer may not have been adapted to the roll inversion in v1.9
+			// Display error message and do not fill actuator outputs
+			// TODO: remove the parameter and this error message in v1.10
+			mavlink_log_critical(&_mavlink_log_pub,
```

Comment:
We can't spam the message this quickly.

Reply:
Ok, I added to the ugliness with a static counter to throttle down the msg rate.

Reply:
We can squash the v19 bits into a single commit to revert later this year.

Reply:
done